R/summary-insertions.R

```diff
@@ -0,0 +1,261 @@
+#' Summary insertions
+#'
+#' Just like [subtotals and headings](SubtotalsHeadings.html), summary statistics can
```

Comment:
Is that a valid link? If you're trying to crossref to another man page, use the `[fun()]` notation.